addon/utils/text.js

```diff
@@ -89,9 +91,8 @@ export function trimLongDataInElement (element) {
   if (textWidth > width) {
     text = trimDataToFit(text, width)
     tooltip = $(element).data('text')
+    $element
```

Comment:
I think rather than putting this in the if condition you should check the new text value against what is currently in the DOM before updating the DOM. Otherwise if a select is renderer to be the width of its container and the browser window is shrunk then expanded the full text may need to become visible again.